hera_cal/vis_clean.py

```diff
@@ -192,20 +172,19 @@ def apply_calibration(self, input_cal, unapply=False):
             input_cal : UVCal, HERACal or filepath to calfits file
             unapply : bool, if True, reverse gain convention to
                 unapply the gains from the data.
-            spw_range : optional spw range to use for filtering.
         """
         # ensure its a HERACal
         hc = io.to_HERACal(input_cal)
         # load gains
         cal_gains, cal_flags, cal_quals, cal_tquals = hc.read()
-        if self.spw_range is not None:
-            for ant in cal_gains:
-                cal_gains[ant] = cal_gains[ant][:, self.spw_range[0]:self.spw_range[1]]
-                cal_flags[ant] = cal_flags[ant][:, self.spw_range[0]:self.spw_range[1]]
-                cal_quals[ant] = cal_quals[ant][:, self.spw_range[0]:self.spw_range[1]]
-            if cal_tquals is not None:
-                for pol in cal_tquals:
-                    cal_tquals[pol] = cal_tquals[pol][:, self.spw_range[0]:self.spw_range[1]]
+        data_freqs = np.logical_and(hc.freqs >= self.data.freqs.min(), hc.freqs <= self.data.freqs.max())
```

Comment:
I'd rename this variable something to clarify that it's a bool like `cal_freqs_in_data`

Reply:
I agree that this suggested variable name is much more readible. I've changed this.